src/frontend/src/components/DesignReviewStageGateModal.tsx

```diff
@@ -0,0 +1,45 @@
+import { useHistory } from 'react-router-dom';
+import { WbsNumber, ChangeRequestType } from 'shared';
+import { useAuth } from '../hooks/auth.hooks';
+import { useCreateStageGateChangeRequest } from '../hooks/change-requests.hooks';
+import { useToast } from '../hooks/toasts.hooks';
+import StageGateWorkPackageModal from '../pages/WorkPackageDetailPage/StageGateWorkPackageModalContainer/StageGateWorkPackageModal';
+import { routes } from '../utils/routes';
+
+interface StageGateModalProps {
+  wbsNum: WbsNumber;
+  modalShow: boolean;
+  handleClose: () => void;
+}
+
+interface FormInput {
+  confirmDone: boolean;
+}
+
+// stage gate modal component (redacted loading and error for this specific case)
+export const StageGateModal: React.FC<StageGateModalProps> = ({ wbsNum, modalShow, handleClose }) => {
+  const auth = useAuth();
```

Comment:
Use useCurrentUser instead, that way we don't have to error handle the user not being there